drivers/SmartThings/philips-hue/src/handlers.lua

```diff
@@ -42,13 +42,13 @@ local function do_switch_action(driver, device, args)
 end
 
 ---@param driver HueDriver
----@param device HueDevice
+---@param device HueChildDevice
 local function do_switch_level_action(driver, device, args)
   local level = st_utils.clamp_value(args.args.level, 1, 100)
   local bridge_device = driver:get_device_info(device:get_field(Fields.PARENT_DEVICE_ID))
 
   if not bridge_device then
-    log.warn("Couldn't get a bridge for light with DNI " .. device.device_network_id)
+    log.warn("Couldn't get a bridge for light with Child Key " .. device.parent_assigned_child_key)
```

Comment:
do we need the `device.device_network_id or device.parent_assigned_child_key` here?

Reply:
Shouldn't need it, all of the handlers will only ever receive bulbs (bridges will be filtered out by now by pseudo-type checks). And lights will never have a DNI with this change.